lib/test/iso/bmff/free.test.ts

```diff
@@ -1,8 +1,8 @@
-import { assert, describe, free, it, parseBox } from './util/box.ts';
+import { assert, describe, free, it, parseBox, type FreeSpaceBox } from './util/box.ts';
 
 describe('free box', () => {
 	it('should correctly parse the box', () => {
-		const box = parseBox('captions.mp4', free, 3);
+		const box = parseBox('captions.mp4', free, 3) as FreeSpaceBox;
```

Comment:
Ideally, `parseBox` should return an array of `FreeSpaceBox[]` based on parser input (e.g. `free`). This will require to extend generic types on `parseBox`